dependents-data/dependents.json

```diff
@@ -0,0 +1,25 @@
+{
+  "direct": [
+    "fastify/fastify",
+    "poolifier/poolifier"
```

Comment:
Is there a way to specify the directory to lint before bundling? 
Because the repo contains a lot of code not shipped in the built bundle as well as not the bundled code.

Reply:
That would require a bit more of a complex setup in the GitHub Actions, doable maybe, but I would do it as a follow up then

Reply:
On the repo side? like an added field to package.json? I would argue that the maintenance burden is pushed on the repo maintainers instead of neostandard ones, maybe with a notification mechanism to repo maintainers in case of failure. 

Without such configuration tunable, poolifier repo will never pass the current test. And I'm pretty sure other repos using bundler will also not pass it.

Reply:
I would like to know more of how to support canary tests of them the best, shall we remove it from here for now and handle it in a follow up issue?

Reply:
Removing poolifier repo until RFC design on how to support it is done is the best move for that PR.

Reply:
Done ✔️